faqs/galaxy/datasets_custom_builds.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Adding Custom Builds to database
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+
+Some tools and functions require that the 'database' attribute that is assigned. A Custom Reference Genome is set up as a Custom Build prior to use. Examples are the tools Featurecounts, Extract Genomic DNA, certain Picard tools, and the functions under Visualization.
+
+Once Custom Build is created, it is added to the list Database/Build: on the dataset 'Edit Attributes' and 'Upload File' tool forms and is available for 'Visualizations'. These can be assigned or used just like any other reference genome.
```

Comment:
Maybe we could move this to the `description:` in the metadata at the beginning of the file? Like it's done [here](https://github.com/galaxyproject/training-material/blob/main/faqs/galaxy/analysis_troubleshooting.md).

Reply:
> Maybe we could move this to the `description:` in the metadata at the beginning of the file? Like it's done [here](https://github.com/galaxyproject/training-material/blob/main/faqs/galaxy/analysis_troubleshooting.md).
Hi! I added  " some tools and functions..." to the description. Do you mean also adding "Once custom build..." to the description?                                                               
Do you suggest shortening the sentence since that sentences are too long?

Reply:
hmm we have this FAQ here: https://training.galaxyproject.org/faqs/galaxy/analysis_add_custom_build.html

I don't know enough to judge how much overlap there is, maybe @shiltemann can help

Reply:
Hi @beatrizserrano  can you please help with this one? What should I do here?

Reply:
We can probably drop this one, the existing "custom builds" FAQ is better.

We should create an issue to link the "custom builds" FAQ to the "custom genome fasta" FAQ and the reverse. Please make that as specific as possible so anyone can do it without having to dig around too much.

Reply:
Ok, then we close this one. Thanks for your contribution @Nurzhamalyrys, please record it in Outreachy so that we count it as done.

Reply:
@beatrizserrano thank you so much ! Going to add it.